blog/serializers/post_serializers.py

```
@@ -23,3 +23,22 @@ class Meta:
+        slug_field="name", read_only=True
+    )
+
+    class Meta:
+        model = Post
+        fields = [
+            "post_id",
+            "category_name",
+            "title",
+            "description",
+            "preview_image",
+        ]
```

Ruff (0.15.5)

[warning] Mutable default value for class attribute

(RUF012)